packages/web-components/src/components/filter-panel/filter-panel.scss

```diff
@@ -0,0 +1,303 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+@import '@carbon/ibmdotcom-styles/scss/globals/vars';
+@import 'carbon-web-components/scss/components/modal/modal';
+@import 'carbon-web-components/scss/components/checkbox/checkbox';
+@import '../../../../styles/scss/components/filter-panel/filter-panel';
+@import 'carbon-components/scss/components/accordion/accordion';
+
+:host(#{$dds-prefix}-filter-panel-modal) {
```

Comment:
There seems to be a lot of styles in the web components scss, wondering how much can be moved to the global styles package, so if for example a non-web components application team could leverage the styles directly from the styles package instead? This would also hopefully keep the styles clean between the web components implementation and what is available in pure SCSS.